PROBot/Scripting/LuaScript.cs

```diff
@@ -1160,11 +1160,40 @@ private bool MoveToMap(string mapName)
         }
 
         // API: Moves to a random accessible cell of the specified rectangle.
-        private bool MoveToRectangle(int minX, int minY, int maxX, int maxY)
+        // Parameters can be four integers, or a table of four integers.
+        private bool MoveToRectangle(DynValue minX, int minY = -1, int maxX = -1, int maxY = -1)

```

Comment:
I'm no c# expert, but I think you can define the same function twice if you have different arguments. This way, you won't have to make the first argument weakly typed.